db/migrate/20201106111550_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name, null: false
+      t.string :last_name, null: false
+      t.string :email, null: false
+      t.references :provider, index: true, null: false, foreign_key: { to_table: :providers }
+
+      t.timestamps
+    end
+
+    add_index :users, %i[email provider_id]
```

Comment:
I'm not sure this is right. It's creating a composite index. Is that what you meant? We should index `email`, `provider_id` is already indexed as a result of line 7.

Reply:
Trello ticket says....
> index on email and provider_id

Line 7 produces https://github.com/DFE-Digital/register-trainee-teachers/pull/147/files#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3R120

Reply:
```suggestion
    add_index :users, :email
```